ibm_was/tests/common.py

```diff
@@ -67,10 +70,10 @@
 }
 
 METRICS_ALWAYS_PRESENT = [
-    'ibm_was.jdbc.create_count',
-    'ibm_was.jvm.free_memory',
-    'ibm_was.servlet_session.life_time',
-    'ibm_was.thread_pools.create_count'
+    'ibm_was.jdbc.percent_used',
```

Comment:
Were these the ones that were just available by default in the new container?

Reply:
Yeah none of these were there

Reply:
😱 Ok thanks!